### ISSUE TITLE
test learning rate decay for LightGBM

### ISSUE THREAD

**Issue by jonas-send-ds** [jonas-send-ds/nmr#23]
see https://github.com/microsoft/LightGBM/issues/2698
